apps/network-alerts/js/attention/attention.js

```diff
@@ -76,6 +77,14 @@ function onVisibilityChange() {
   }
 }
 
+function render() {
+  renderForm();
```

Comment:
nit: maybe ```renderForm``` can accept ```params``` (or even better ```title``` and ```body``` as separate parameters) so that we call ```Utils.parseParams()``` only once in ```render``` and only ```render``` will know about our hacky param source :) ?